workloads/kube-burner/workloads/node-vm-density/vm.yml

```diff
@@ -0,0 +1,50 @@
+apiVersion: kubevirt.io/v1
+kind: VirtualMachine
+metadata:
+  labels:
+    name: {{.JobName}}
+  name: {{.JobName}}-{{.Iteration}}
+spec:
+  running: true
+  template:
+    metadata:
+      labels:
+        kubevirt-vm: {{.JobName}}-{{.Iteration}}
+    spec:
+      domain:
+        cpu:
+          cores: 1
+          sockets: 1
+          threads: 1
+        devices:
+          disks:
+            - disk:
+                bus: virtio
+              name: node-vm-density
+        machine:
+          type: pc-q35-rhel8.4.0
+        resources:
+          requests:
+            memory: 256Mi
+            cpu: 100m
+          limits:
+            cpu: 100m
```

Comment:
Why the hard wired limit?

Reply:
It's a density test that is meant to stress QPS, in order to have as many VMs as possible I have set the limit to the min requirement.